packages/e2e-tests/specs/editor/various/invalid-block.test.js

```diff
@@ -94,9 +94,6 @@ describe( 'invalid blocks', () => {
 
 		// Give the browser time to show the alert.
 		await page.evaluate( () => new Promise( window.requestIdleCallback ) );
-
-		expect( console ).toHaveWarned();
-		expect( console ).toHaveErrored();
```

Comment:
The console logs were triggered because the block was considered invalid, it's now valid, I don't see why the block should be invalid here.

Reply:
I think this looks good. Expectation below remains unchanged and that's what matters.